doc/api/synopsis.md

```diff
@@ -9,9 +9,62 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Commands displayed in this document are shown starting with `$` or `>` 
+to replicate how they would appear in a user's terminal. 
+Do not include the `$` and `>` character.
+They are there to indicate the start of each command. 
+
+There are many tutorials and examples that follow this 
+convention: `$` or `>` for commands run as a regular user, and `#` 
+for commands that should be executed as an administrator. 
+
+Lines that don’t start with `$` or `>` character are typically showing 
+the output of the previous command.
+
+Firstly, make sure to have downloaded Node.js from [Node.js Official website][].
+
+To install Node using a package manager, see [this guide][].
+
+
+Now, create an empty project folder called `projects`, navigate into it:
+Project folder can be named base on user's current project title but
+this example will use `projects` as the project folder. 
+
+UNIX :
+
+```console
+$ mkdir ~/projects
+$ cd ~/projects
+```
+Windows CMD:
+```console
+> mkdir %USERPROFILE%\projects
+> cd %USERPROFILE%\projects
+```
+
+Windows PowerShell:
+
+```console
+> mkdir $env:USERPROFILE\projects
+> cd $env:USERPROFILE\projects
+```
+
+Next, create a new source file in the `projects` folder
+and call it `hello-world.js`.
+
+If filename has more than one word, use a hyphen(`-`) or 
+an underscore(`_`) to separate them for simplicity and avoid using 
+the space character in file names. 
+
+For example, use `hello-world.js` rather than `hello world.js`.
+Node.js files always end with the `.js` extension. 
```

Comment:
I personally do not feel like this limitation should be enforced, no matter that it is good style. The point for me would be to educate the user by telling why and that is missing here.
So instead of these two paragraphs I would write e.g.:

```
In Node.js it is considered good style to use hypens (`-`) or underscores (`_`) to separate
multiple words in filenames.
```

Besides that not only Node.js files end with `.js` but any JavaScript file. Is that part really necessary to be documented?